servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,13 @@
    */
   LRU,
   /**
+   * Evict old blocks in certain StorageDir by LRU. The main difference between PartialLRU and
+   * LRU is that LRU choose old blocks among several StorageDirs until one StorageDir
+   * satisfies the request space, but PartialLRU select one StorageDir first and evict old
+   * blocks in certain StorageDir by LRU
+   */
+  PARTIAL_LRU,
```

Comment:
@Yours-shupeng hi, shupeng, I haven't read the code, through the description of javadoc, the logic of PARTIAL_LRU is the same when passing a determined BlockStoreLocation(other than anyTier or anyDirInTier) to current LRUEvictor?